src/libAtomVM/opcodesswitch.h

```diff
@@ -1423,9 +1444,9 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
             case OP_ALLOCATE_HEAP: {
                 int next_off = 1;
                 int stack_need;
-                DECODE_INTEGER(stack_need, code, i, next_off, next_off);
+                DECODE_ALLOCATOR_LIST(stack_need, code, i, next_off, next_off);
```

Comment:
Which test covers this case? I'm not sure `stack_need` can be an extended allocator list with OTP24.

Reply:
removed